2-Regression/translations/README.az.md

```diff
@@ -0,0 +1,39 @@
+# Maşın Öyrənməsi üçün reqressiya modelləri
+## Regional mövzu: Şimali Amerikada balqabaq qiymətləri üçün reqressiya modelləri 🎃
+
+Şimali Amerikada balqabaqlar tez-tez Hellouin bayramı üçün qorxunc üz formasında çərtilirlər. Gəlin bu tərəvəz haqqında maraqlı faktları öyrənək!
+
+![jack-o-lanterns](./images/jack-o-lanterns.jpg)
+><a href="https://unsplash.com/@teutschmann?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Beth Teutschmann</a> tərəfindən çəkilmiş şəkil. Mənbə: <a href="https://unsplash.com/s/photos/jack-o-lanterns?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+
+## Nə öyrənəcəksiniz
+
+[![Reqressiyaya giriş](https://img.youtube.com/vi/5QnJtDad4iQ/0.jpg)](https://youtu.be/5QnJtDad4iQ "Reqressiya giriş videosu - İzləmək üçün klikləyin!")
+> 🎥 Bu dərsə sürətli giriş videosu üçün yuxarıdakı şəkilə klikləyin.
+
+Bu bölmədəki dərslər maşın öyrənməsi konteksti çərçivəsində reqressiyaların növlərini əhatə edir. Reqressiya modelləri dəyişənlər arasındakı _əlaqəni_ təyin etməyə kömək edir. Bu model tipi dəyişənlər arası əlaqəni data nöqtləri vasitəsilə analiz edərək üzə çıxarır və bunun sayəsində uzunluq, temperatur və ya yaş kimi dəyərləri təxmin edə bilir.
```

Comment:
konteksti -> mövzusu?

Reply:
I think no. In a daily usage, most of us using "kontekst çərçivəsində" instead of "mövzu çərçivəsində". Also they are not the same